tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { of } from 'rxjs';
+import { DataService } from '../../src/app/core/services/data.service';
+import { GithubService } from '../../src/app/core/services/github.service';
+import { autoSpy } from '../auto-spy';
+import { csvString, dataFile, dataFileTeamStructure } from '../constants/data.constants';
+
+describe('DataService', () => {
+  it('when getDataFile is called it should return the correct data', done => {
+    // arrange
+    const { build, githubService } = setup().default();

```

Comment:
Is there a need to call `default`, since `setup` seems to return the same object?

Reply:
I will leave it in there, as it does not affect the tests if removed either.

Reply:
Thanks for the response. Just want to get more clarity - is there a benefit to keeping the `default()` call if it is redundant?
My concern is that redundant code may make it harder for other developers to understand the tests.

Reply:
@JunWei96, may we get your opinion here, on whether the usage of  `default` should be kept?

Reply:
@geshuming would there be a drawback to removing the usage of `default` ?

Reply:
Theres no drawback. I will remove the default() method to make the tests cleaner